ruff.toml

```diff
@@ -59,6 +63,13 @@ ignore = [
 [lint.flake8-annotations]
 ignore-fully-untyped = true
 
+[lint.flake8-pytest-style]
+parametrize-names-type = "csv"
```

Comment:
I went with `"csv"` because it's the most used in the project currently. But I think I'd prefer the default of `"tuple"` See https://docs.astral.sh/ruff/rules/pytest-parametrize-names-wrong-type/ & https://docs.astral.sh/ruff/settings/#lint_flake8-pytest-style_parametrize-names-type

Reply:
It should be OK to change to `tuple` if we have the linter to enforce it in future PRs.
Can `ruff` automatically fix that?
If I understand correctly the maintenance policy in the repository is that implicit config is better than explicit config, so it would be nice if we can rely on defaults.

Reply:
It can autofix it, it's simply marked as an "unsafe fix". Happy to provide a follow-up PR ! (unless you want me to apply it here, there's 41 hits)